src/Microsoft.AspNetCore.Server.IISIntegration/Server/IISHttpServer.cs

```diff
@@ -96,7 +96,15 @@ private static NativeMethods.REQUEST_NOTIFICATION_STATUS OnAsyncCompletion(IntPt
             return NativeMethods.REQUEST_NOTIFICATION_STATUS.RQ_NOTIFICATION_PENDING;
         }
 
-        private static void CompleteRequest(IISHttpContext context)
+        private static bool OnClientDisconnect(IntPtr pvManagedHttpContext)
+        {
+            var context = (IISHttpContext)GCHandle.FromIntPtr(pvManagedHttpContext).Target;
+            context?.OnClientDisconnect();
+            return true;
+        }
+
```

Comment:
nit: extra space

Reply:
I removed that, not sure why it is still showing up.